README.md

```diff
@@ -96,6 +96,22 @@ Yes, it's really all you need to get started as you can see in this live and int
 
 [![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/4j7m47vlm4)
 
+## Should I start with v1-beta?
+
+Yes, you should. We often get this question:
+
+> Should I start with v1-beta? But beta is a beta, so it's not a final product and I'm not guaranteed anything.
+
+People are starting projects with v0.x which given the quality and stability of v1 they shouldn't be. The v1-beta effort started in May 2016.
+There are lots of us using v1-beta in production already with no problems and the odd breaking change is less hassle than upgrading from v0.x to v1-beta.
+They are just creating extra work for themselves as they will have to transition at some point.
```

Comment:
This sentence needs to go before "There are lots of us" (or the @kgregory's suggested change.